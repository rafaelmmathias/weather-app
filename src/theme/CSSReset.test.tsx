import renderer from "react-test-renderer";
import { CSSReset } from "./CSSReset";

describe("Component - CSS Reset", () => {
  it("should match snapshot with default values", () => {
    const tree = renderer.create(<CSSReset />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
