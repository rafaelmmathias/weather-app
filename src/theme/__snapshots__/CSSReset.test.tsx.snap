// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - CSS Reset should match snapshot with default values 1`] = `null`;
